docs/basics/mutating-data.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Mutations
+order: 2
+---
+
+## Mutate your data
+
+Now that we know how to query our data we'll also need to know
```

Comment:
Might be worth keeping docs isolated from other sections given how pages are separate. Straight to the point might be more useful.

> # Mutations
>Mutations can be triggered in Urql in one of two ways
>
> - directly via the client using `executeMutation`
> - indirectly using an adapter - (e.g. `useQuery`)
> 
> ## Client (executeMutation)
> ...
>
> ## React/Preact (useMutation)
>
> ...
>
> ## JQuery ($mutation)
>
> ...
>